packages/neuron-ui/src/components/SendFromMultisigDialog/hooks.ts

```diff
@@ -118,28 +140,76 @@ export const useSendInfo = ({
   const dispatch = useDispatch()
   const [errorMessage, setErrorMessage] = useState('')
   useEffect(() => {
-    if (outputErrors.some(v => v.addrError || v.amountError)) {
-      return
-    }
     clearTimeout(generateTxTimer)
     setErrorMessage('')
-    const validSendInfoList = sendInfoList.filter(v => v.address && v.amount)
+    const validSendInfoList = sendInfoList.filter(
+      (v, i) => v.address && v.amount && !outputErrors[i].addrError && !outputErrors[i].amountError
+    )
     generateTxTimer = setTimeout(() => {
       dispatch({
         type: AppActions.UpdateGeneratedTx,
         payload: null,
       })
       if (validSendInfoList.length) {
-        generateMultisigTxWith({
+        generateMultisigTxWith(generateMultisigTx)({
           sendInfoList: validSendInfoList,
           setErrorMessage,
           multisigConfig,
           dispatch,
           t,
+          isMainnet,
         })
       }
     }, 300)
-  }, [sendInfoList, setErrorMessage, multisigConfig, dispatch, t, outputErrors])
+  }, [sendInfoList, setErrorMessage, multisigConfig, dispatch, t, isMainnet, outputErrors])
+  const [isSendMax, setIsSendMax] = useState(false)
+  const onSendMaxClick = useCallback(
+    e => {
+      const {
+        dataset: { isOn = 'false' },
```

Comment:
What does `isOn` mean?

Reply:
It equal with the `isSendMax`

Reply:
What does `setIsSendMax(isOn === 'false')` mean if `isSendMax` equals to `isSendMax`

Reply:
Ah, I understand it, when `isOn` is `false`, by clicking the button, `isSendMax` is set to `true`